connector-email/src/test/java/com/redhat/cloud/notifications/connector/email/processors/bop/BOPRequestPreparerTest.java

```diff
@@ -52,7 +52,7 @@ void testProcess() {
         final Map<String, Object> headers = exchange.getMessage().getHeaders();
         assertEquals(HttpMethods.POST, headers.get(Exchange.HTTP_METHOD));
         assertEquals("/v1/sendEmails", headers.get(Exchange.HTTP_PATH));
-        assertEquals("application/json", headers.get(Exchange.CONTENT_TYPE));
+        assertEquals("application/json; charset=utf-8", headers.get(Exchange.CONTENT_TYPE));
```

Comment:
Charset was missing, it was not a problem with previous Quankus and dependencies versions, but without it BOP complains now.